components/AffirmationCarousel.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { AffirmationCard } from '@/components/AffirmationCard';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+} from "@/components/ui/carousel"
+
+
+interface AffirmationCarouselProps {
+  initialAffirmations: string[];
+}
+
+export function AffirmationCarousel({
+  initialAffirmations,
+}: AffirmationCarouselProps) {
+  return (
+    <Carousel>
+      <CarouselContent className="items-center">
+        {Array.from(initialAffirmations).map((text, index) => (
```

Comment:
Why `Array.from` here?
```suggestion
        {initialAffirmations.map((text, index) => (
```